tests/components/automation/test_init.py

```diff
@@ -359,41 +359,54 @@ def test_services(self):
 
         automation.toggle(self.hass, entity_id)
         self.hass.block_till_done()
+        self.hass.bus.fire('test_event')
```

Comment:
I will give time for event loop to process things with `self.hass.block_till_done()` while `automation.toogle` don't blocking untill service call is finish and `self.hass.block_till_done()` work some time bad on travis. But you have more benefit on unittest and could have a better idee.